blocks/v2-images-grid/v2-images-grid.css

```diff
@@ -0,0 +1,95 @@
+:root {
+  --v2-images-grid-gutter: 8px;
+  --v2-images-grid-big-height: 168px;
+  --v2-images-grid-small-height: 125px;
+  --v2-images-grid-padding-space: 24px;
+}
+
+.redesign-v2 .v2-images-grid-container .v2-images-grid-wrapper {
+  padding: 0 100px;
```

Comment:
We need to remove the `padding: 0 200px` from styles and use `max-width: var(--wrapper-width)` instead. So all the blocks follow the same size.

Reply:
Changed that